tool_path_planner/src/surface_walk_raster_generator.cpp

```diff
@@ -48,8 +48,6 @@ namespace tool_path_planner
  */
 struct PathEndPoints
 {
-  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
Just to clarify, this is being removed because Eigen::Vector3d is non-vectorizable and does not require it, correct?

It may be better to comment it out and leave a comment to that effect.  Otherwise, someone well-intentioned may add it back in next month.

```
// EIGEN_MAKE_ALIGNED_OPERATOR_NEW // Not necessary, Eigen::Vector3d is non-vectorizable.
```

Reply:
I think `Vector3d` is technically vectorizable like most (all?) other `Eigen` matrix types, but it doesn't have the alignment issues because it's size is not a multiple of 16 bytes ([reference](https://eigen.tuxfamily.org/dox/group__TopicFixedSizeVectorizable.html)). This is just me trying to be technically correct; it doesn't hurt anything if this macro is here

Reply:
By "vectorizable", I meant "fix-sized vectorizable" as defined in that reference, not "able to be placed in a std::vector" or some other definition.  I agree with removing the macro, as it is not necessary here, but was just speculating that sometimes unnecessary code is best left commented-out rather than removed.  I have seen some things creep in to code over and over again on past projects.  Not a big deal.

Reply:
I agree based on the documentation, but given how much trouble we have had with Eigen I would feel better with it staying since it includes Eigen.